web/viewer.html

```diff
@@ -334,6 +347,8 @@
               </div>
               <div id="toolbarViewerRight">
                 <div id="editorModeButtons" class="splitToolbarButton toggled" role="radiogroup">
+                  <button id="editorHighlight" class="toolbarButton" hidden="true" disabled="disabled" title="Highlight" role="radio" aria-checked="false" aria-controls="editorHighlightParamsToolbar" tabindex="37" data-l10n-id="pdfjs-editor-highlight">
```

Comment:
The `tabindex` is incorrect here, it should be *lower* than the FreeText-button that follows it.
Perhaps this button should have `tabindex="31"` now, and the following ones be re-indexed?